db/connect.rb

```diff
@@ -3,6 +3,9 @@
   when :test
     require 'sqlite3'
     DB = Sequel.connect('sqlite://test/stations_bot-test.db')
+  when :development
+    require 'mysql2'
+    DB = Sequel.connect(ENV['DATABASE_URL'])
```

Comment:
We should probably clean up these database connections, too. I won't block this PR on it, though.